### ISSUE TITLE
Tailwind css overriding global website stylesheet

### ISSUE THREAD

**Issue by SophieNikki** [doublethreedigital/cookie-notice#22]
In the notice.blade.php file, at the bottom of the file the stylesheet is imported. But this stylesheet with tailwind is overriding a lot of my bootstrap and own styling at my website. I'm not using Tailwind there.

Is there a possibility to remove or override the tailwind that is rendered in the plugin? Or get it only used on the cookie consent and not globally on the whole site?

**Comment by damcclean** [comment]
Hmm, it's weird that the Tailwind styles are overwriting your thing. The tailwind classes should be getting [prefixed](https://github.com/doublethreedigital/cookie-notice/blob/master/tailwind.config.js#L2) and any un-used styles should be purged and removed from the source css file.

I'll have a greater look tonight when I'm free to investigate. Maybe the CSS hasn't been compiled properly or something.

**Comment by SophieNikki** [comment]
Thanks!

**Comment by damcclean** [comment]
There we go! Tagged another bug fix release, [v3.0.5](https://github.com/doublethreedigital/cookie-notice/releases/tag/v3.0.5). 

I had `@tailwind base` in my stylesheet which meant Tailwind included it's Normalize presets which I think is what'll be causing the issue with your site.

You should be able to update by running `composer install` and re-publishing Cookie Notice's CSS files with `php artisan vendor:publish --tag=cookie-notice-assets`

**Comment by SophieNikki** [comment]
I still have the same issue that it is overriding my css. I have deleted the vendor folder, then ran composer install and the php artisan. But still the same outcome..
I fixed it for now by putting my css (bootstrap and my custom styling) at the bottom of the page, at the end of the body so that it will override the cookie consent. 

I tried some stuff locally but I can't find another solution to fix it.. 

![Screenshot 2020-10-01 at 11 23 38](https://user-images.githubusercontent.com/72135050/94791963-a3bfb000-03d8-11eb-9c3e-6acf59614ed9.png)

**Comment by damcclean** [comment]
Hmm, that's weird. I'll investigate more this evening. I've re-opened the issue.

**Comment by damcclean** [comment]
Are you sure you're using the most up-to-date release of Cookie Notice, [v3.0.5](https://github.com/doublethreedigital/cookie-notice/releases/tag/v3.0.5). You should be able to check the version by running `php please support:details`. 

It may have been a Composer caching issue that it didn't pick up the latest version, you can clear that by running `composer clear-cache`

The latest version doesn't have any of the reset stuff, like the ones in the screenshot you added above, [see the file](https://github.com/doublethreedigital/cookie-notice/blob/master/resources/dist/css/cookie-notice.css) here.

Let me know how you get on.

**Comment by damcclean** [comment]
I'm closing this issue now due to inactivity. If you're still running into this issue comment below and I can re-open the issue.

**Comment by SophieNikki** [comment]
Hey @damcclean, 

I am sorry for the late response. I got pulled away to work on another project and didn't have time to check this out. I need to reinstall the plugin on the other project to check if it's working there, but it seems to be perfectly working on the project I am working on now. So that's great! 

Also, I found a typo in the Readme at the 'Overriding the cookie notice' part. In the code it says 'publush' instead of 'publish'

Thanks for your help! :)
